vanilla-framework/vanilla-framework-react
# Add code snippets

# Done
- Add code snippet pattern

# QA
- Pull code
- Run `yarn run build` and `yarn run serve`
- Verify the Code Snippet pattern

# Fixes
Fixes #29
